config/prometheus/templates/deployment.yaml

```diff
@@ -8,6 +8,9 @@ metadata:
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: Helm
     app: prometheus
+    {{- with .Values.commonLabels }}
+    {{- toYaml . | nindent 4 }}
+    {{- end }}
```

Comment:
Am I correct that `.Values.commonLabels` is totally separate from the setup task labels?

Reply:
Correct. There might be some clever bits we could do in the entry point to try to make it happen automatically, but not sure it was worth the effort / complexity.